wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -581,6 +581,38 @@ public WrapperCommand withName(String name) {
     return wrapper;
   }
 
+  /**
+   * Throws an error if a parallel group already shares
+   * one or more requirements with a command
+   * that will be added to it.
+   *
+   * @param parallelGroup The parallel group command.
+   * @param toAdd The command that will be added to the parallel group.
+   */
+  protected void ensureDisjointRequirements(Command parallelGroup, Command toAdd) {
+    var sharedRequirements = new HashSet<>(parallelGroup.getRequirements());
```

Comment:
```suggestion
  protected void ensureDisjointRequirements(Command toAdd) {
    var sharedRequirements = new HashSet<>(getRequirements());
```
Might as well use that it's non-static.